lib/cocina/models/file_set.rb

```diff
@@ -17,11 +17,11 @@ class Identification < Dry::Struct
 
       # Structural sub-schema for the FileSet
       class Structural < Dry::Struct
-        attribute :contains, Types::Strict::Array.of(Types::Coercible::String).meta(omittable: true)
+        attribute :contains, Types::Strict::Array.of(Cocina::Models::File).meta(omittable: true)
```

Comment:
Can `Cocina::Models::` be dropped from lines 20 and 24? The explicit namespace does not appear to be required in the changing to dro.rb above.

Reply:
I just don't want anyone to think this is `::File`, so I'm being explicit here.

Reply:
Aha. Makes sense.